examples/simple/index-webpack.html

```diff
@@ -1,19 +1,19 @@
 <!DOCTYPE html>
-<html>
-
-<head>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Why did you change that?

Reply:
Because the old values didn't provide any value AFAIK. Besides, `minimum-scale` can be an accessibility issue for browsers that still respect them (those I use don't actually)